publish/models/asset.py

```diff
@@ -105,3 +105,22 @@ def from_girder(cls, version: Version, girder_file: GirderFile, client: GirderCl
             # The actual upload of blob occurs when the asset is saved
             asset.save()
         return asset
+
+    @classmethod
+    def get_path(cls, path_prefix: str, qs: str) -> set:
```

Comment:
Isn't `qs` a list of str?

Reply:
Yes, fixed in https://github.com/dandi/dandi-publish/pull/111/commits/23c77858bb09f292dfddf55ecc02b8e80331676b